.ci/bump-stack-release-version.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+#
+# Given the stack version this script will bump the release version.
+#
+# This script is executed by the automation we are putting in place
+# and it requires the git add/commit commands.
+#
+# Parameters:
+#	$1 -> the minor version to be bumped. Mandatory.
+#	$1 -> the major version to be bumped. Mandatory.
+#
+set -euo pipefail
+MSG="parameter missing."
+VERSION_RELEASE=${1:?$MSG}
+VERSION_DEV=${2:?$MSG}
+
+OS=$(uname -s| tr '[:upper:]' '[:lower:]')
+
+if [ "${OS}" == "darwin" ] ; then
+	SED="sed -i .bck"
+else
+	SED="sed -i"
+fi
+
+MAJOR_MINOR_RELEASE=$(cut -d '.' -f 1,2 <<< "$VERSION_RELEASE")
+
+echo "Update stack with versions ${VERSION_RELEASE} and ${VERSION_DEV}"
+${SED} -E -e "s#'[0-9]+\.[0-9]+-SNAPSHOT'#'${MAJOR_MINOR_RELEASE}-SNAPSHOT'#g" .ci/Jenkinsfile
```

Comment:
I think this will update all the versions in the Jenkinsfile, not the ELASTIC_AGENT_STALE_VERSION one.

What we already do in the bump-stack-version is:
```
${SED} -E -e "s#(name: 'ELASTIC_AGENT_VERSION', defaultValue: ')[0-9]+\.[0-9]+\.[0-9]+(-[a-f0-9]{8})?#\1${VERSION}#g" $FILE
```

Reply:
Wait, looking at the sed regex, it will match versions with two groups, right?